components/tests/integration_test.cpp

```diff
@@ -46,18 +58,16 @@ int authenticate_json(const std::string& json_file, const bool compute_validatio
     in_args["root_path"] = json_file;
 
 
-    if (!compute_validation &&
-        (!in_args["tests"].contains("vertices") || !in_args["tests"].contains("edges") ||
-         !in_args["tests"].contains("faces") || !in_args["tests"].contains("tetrahedra"))) {
+    if (compute_validation && !contains_results(in_args)) {
         spdlog::error("JSON file missing vertices edges faces or tetrahedra or meshes key. Add a * "
                       "to the beginning of filename to allow appends.");
         return 2;
     }
 
-    in_args["settings"] = R"({
-        "log_level": 5,
-        "opt_log_level": 5
-        })"_json;
+    //in_args["settings"] = R"({
```

Comment:
because it's a pain having unit tests hide data - i'd rather have someone run the entire suite -> run the tests that fail with manual control over logging level than have the test control output levels manually.